html/feed/feed.html

```diff
@@ -6,36 +6,148 @@
+      <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
+      <link rel="stylesheet" href="/html/feed/feed.css">
+
+    <script src = "../utils/logout.js" type = "module"></script>
+    <script src="/html/feed/feed.js" type="module"></script>
 </head>
-<body class="p-3 m-4 border-0 bd-example bg-secondary bg-opacity-50">
 
-      <ul class="nav justify-content-end">
+<body class="p-3 m-4 border-0 bd-example bg-dark">
+ <header> 
+   <nav class="navbar navbar navbar-expand-lg navbar-dark bg-dark m-3">
 
-        <a class="navbar-brand">
+     
+        <a class="navbar-brand" href="/html/feed/feed.html">
           <img src="/img/mouselogo.jpg" class="rounded-circle mx-auto d-block img-fluid"  alt="mouse with glasses" width="30" height="24">
         </a>
+
+        <button class =" navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target ="#navbarNav" aria-controls ="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <i class ="fas fa-bars"></i>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+          <a class="nav-link" href="/html/feed/feed.html">Nerdsnett</a>
+          </li>
+          <li class="nav-item">
+          <a class="nav-link" href="/html/profile/profile.html">My profile</a>
+          </li>
+          <li class="nav-item">
+            <button class = " nav-link" tabindex="-1" id="logOut">Log out</button>
+          </li>
+        </ul>
+    </nav>
+ </header>
+
+ <main class="container custom-container" id="feed-container">
+
+  <h1 class="text-white">Nerdsnett</h1>
+    <nav class="nav-container mb-3">
+      <ul class="nav nav-pills">
         <li class="nav-item">
-          <a class="nav-link link-dark" href="/html/feed.html">Nerdsnett</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link link-dark" href="/html/profile.html">My profile</a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link disabled">log out</a>
+          <a class="nav-link" id="filter-tab" data-bs-toggle="pill" href="#filter">Filter</a>
+        </li><li class="nav-item">
+          <a class="nav-link" id="search-tab" data-bs-toggle="pill" href="#search">Search</a>
+        </li><li class="nav-item">
+          <a class="nav-link" id="create-post-tab" data-bs-toggle="pill" href="#create-post">Create Post</a>
         </li>
-        
       </ul>
 
+      <!--sort content
+      <div class="tab-content mt-4">
+        
+        <div class="tab-pane fade show active" id="sort" role="tabpanel" aria-labelledby="sort-tab">
+          <select id="sortSelect" class="form-control">
+            <option value="latest">Latest</option>
+            <option value="oldest">Oldest</option>
+             Add more sort options as needed -->
+             
+        </select>
+        </div>
```

Comment:
half a div here

Reply:
this one is a part of the sort <div> too, its commented out now.